actionpack/test/dispatch/server_timing_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "abstract_unit"
+
+class ServerTimingTest < ActionDispatch::IntegrationTest
+  class TestController < ActionController::Base
+    def index
+      head :ok
+    end
+
+    def show
+      ActiveSupport::Notifications.instrument("custom.event") do
+        sleep 0.001
```

Comment:
Since we really don't care about what we are doing here we can just use `true` instead

Reply:
Done.